src/components/PopupWithForm.js

```diff
@@ -7,13 +7,8 @@ export default class PopupWithForm extends Popup {
     this._handleFormSubmit = handleFormSubmit;
     this._submitButton = this._popupElement.querySelector("modal__button-save");
     this._inputList = [...this._popupForm.querySelectorAll("modal__input")];
-    // this._formElement = this._popupElement.querySelector(".modal__form");
   }
 
-  // close() {
-  //   super.close();
-  // }
-
```

Comment:
**Could be improved**

You can make ` setInputValues` in class `PopupWithForm`, which can insert data into inputs:

```js

  setInputValues(data) {
    this._inputList.forEach((input) => {
      // Here you insert the `value` by the `name` of the input
      input.value = data[input.name];
    });
  }



```

So, this way you’ll not have to search the inputs of the profile in `index.js`